packages/vm/lib/runBlock.ts

```diff
@@ -149,17 +149,26 @@ export default async function runBlock(this: VM, opts: RunBlockOpts): Promise<Ru
   }
 
   // Checkpoint state
-  await state.checkpoint()
+  // (only do checkpointing when txs are executed)
+  // TODO: determine if checkpointing is needed in a block
+  // context at all or CPs in runTx() are sufficient
```

Comment:
Yeah, we want this - if we run a bad block, we want to rollback the DB to the version how it was before running the bad block.

Reply:
Ok, have removed the comment part. 😛